docs/build/vm/create/rust-vm.md

```diff
@@ -19,9 +19,8 @@ payload of data.
 ## Prerequisites
 
 - Install the latest stable version of Rust using [`rustup`](https://www.rust-lang.org/tools/install).
-- Bookmark and review the [avalanche-types](https://github.com/ava-labs/avalanche-types-rs) GitHub
-  repository specifically the traits and helpers defined in the
-  [`subnet/rpc`](https://github.com/ava-labs/avalanche-types-rs/tree/main/src/subnet/rpc) mod.
+- Bookmark and review the [avalanche-rs](https://github.com/ava-labs/avalanche-rs) GitHub
+  repository, specifically the subnet traits and helpers defined in the `avalanche-types` crate
 - For developers new to Rust please visit the free online book [The Rust Programming
   Language](https://doc.rust-lang.org/book/).
```

Comment:
```suggestion
- Bookmark and review the [avalanche-rs](https://github.com/ava-labs/avalanche-rs) GitHub
  repository, specifically the subnet traits and helpers defined in the `avalanche-types` crate
- For developers new to Rust please visit the free online book [The Rust Programming
  Language](https://doc.rust-lang.org/book/).
  
  :::note
  The example VMs in these tutorials are based on
  [avalanche-types-rs](https://github.com/ava-labs/avalanche-types-rs), a predecessor of
  the [avalanche-rs](https://github.com/ava-labs/avalanche-rs) repository that is now the
  accepted standard. Directory locations will vary.
  :::
  
```
TimestampVM and other referred VMs still use avalanche-types-rs and they still haven't been updated so users would have to navigate between using the two different crates. I've added a note for clarification, but rest assured these docs are going to be updated entirely in the near future.